### DISCUSSION TITLE
Good first issue for new contributors

### DISCUSSION THREAD

**Discussion by salihozdemir** [Trendyol/baklava#708] [Q&A] [answered]
I would like to contribute to this great project. Is there a suitable issue to start with? Maybe you can use the label "good first issue" for this kind of issues. Thanks.

**Comment by leventozen** [comment, marked as answer]
Hi @salihozdemir, that's very good idea. We will label our issues accordingly. I am curious what do you think about online workshops/pairing sessions about baklava?

**Reply by salihozdemir** [reply]
Hey @leventozen,  I would love to join and learn more about baklava. But what do yo expect from me? I am not a baklava expert. So, I don't want to disappoint your expectations.

**Reply by leventozen** [reply]
We as baklava team are thinking about organizing workshops about contributing to Baklava, if the community wants this. I wanted to hear your opinion about this 👋
